requirements.txt

```diff
@@ -2,7 +2,7 @@ psutil==5.8.0
 requests~=2.27.1
 grpcio==1.43.0
 protobuf==3.19.4
-docker==5.0.0
+docker~=5.0.0
```

Comment:
Why not upgrade to 5.0.3 as it is in Pineapple? Or downgrade Pineapple? @Jongy

Reply:
This is a utils lib so let the users i.e gprofiler/pineapple pick the exact version

Reply:
Kk